app/clients/lightning_network_client.rb

```diff
@@ -1,9 +1,10 @@
 class LightningNetworkClient
   include HTTParty
-  LN_BASE_URL = ENV.fetch('LN_BASE_URL')
-  INVOICE_MACAROON = ENV.fetch('INVOICE_MACAROON')
+  LND_HOST = ENV.fetch('LND_HOST')
+  LND_PORT = ENV.fetch('LND_PORT')
+  INVOICE_MACAROON = ENV.fetch('LND_INVOICE_MACAROON')
 
-  base_uri LN_BASE_URL
+  base_uri "https://#{LND_HOST}:#{LND_PORT}/v1"
```

Comment:
Buena, tenía mis dudas de que esto funcionara, pero veo que en la linea 10 está el `verify: false`

Reply:
Yes, la idea era q en development no tuviéramos que tener muchas cosas raras. Cómo el cert y los macaroons.
Igual nos que pendiente usar el cert en producción. :)

Reply:
Sí no es muy lindo tenerlo así en producción... Voy a crear un issue para que no se nos olvide.

Lo otro, sobre lo de minio.. No creo que sea necesario porque `ActiveStorage` por defecto guarda las cosas en el sistema de archivos local y para desarrollar funciona bien. Así que estaría listo para el merge y poder avanzar como 🚀

Reply:
Aunque pensándolo bien.. no será más fácil cambiar `LND_HOST` y `LND_PORT` por `LND_URL`? (o hacer `base_uri LND_URI || "https://#{LND_HOST}:#{LND_PORT}/v1"`)

Reply:
- me gusta tener el host y port, como que nos abstrae de la "forma" del url...
- tambien me gusta que sea consistente con los otros servicios como db y redis
- ademas el port en desarrollo se lo preguntamos a docker tambien de una manera media estandar..

Reply:
Sí, te entiendo y estoy de acuerdo... Lo que estaba tratando de encontrar es una forma en la cual se pueda definir a través de una configuración del ambiente si esa conexión usará http o https.

¿Y hacer algo como
```ruby
LND_SSL_VERIFY = ENV.fetch('LND_SSL_VERIFY', true)

...

check_success self.class.get("/invoices", headers: headers, verify: LND_SSL_VERIFY)
```
?

Así podemos agregar eso en el `.env.development` y modificarlo en producción hasta que lo solucionemos.

Si no, vamos con el merge nomás, no me quedaría pegado en esto. 👍

Reply:
Claro, pero LND solo expone https, no http, asique igual creo que esta bien que quede hardcodeada la forma del url con `https://`

Igual esta buena la idea de dejar configurable el verify

Reply:
Te tinca algo asi?

En vez de `LND_SSL_VERIFY` tener un `LND_SSL_CERT` con el cert...

Reply:
```ruby
...
pem ENV["LND_SSL_CERT"] if verify?

---

check_success self.class.get("/invoices", headers: headers, verify: verify?)

---

def verify?
  ENV["LND_SSL_CERT"].present?
end
```

Reply:
❤️ like it

Así dejamos listo este tema y después sólo nos queda actualizar la configuración en producción.

Reply:
Dejemos el problema de produccion con cert para el futuro